osbrain/__init__.py

```diff
@@ -1,4 +1,18 @@
 import os
+import sys
+from pathlib import Path
+
+# While Python 3.4 is supported...
+if sys.version_info < (3, 5):
+    def mkdir(directory, exist_ok=True, parents=True, **kwargs):
```

Comment:
You are right. I thought about implementing those parameters to create a compliant `.mkdir` method in Python 3.4 `Path().mkdir`. But as we do not require that at the moment, I thought we could always assume `parents=True` and `exist_ok=True`.

I will remove them. Thanks! :blush: